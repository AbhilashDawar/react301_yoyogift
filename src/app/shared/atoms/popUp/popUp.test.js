import React from 'react';
import AppPopUp from './index.jsx';
import renderer from 'react-test-renderer';

test('AppPopUp Component', () => {
    const tree = renderer.create(
        <AppPopUp />
    ).toJSON();
    expect(tree).toMatchSnapshot();
});