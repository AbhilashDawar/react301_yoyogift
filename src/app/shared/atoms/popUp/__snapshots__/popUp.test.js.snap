// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppPopUp Component 1`] = `<div />`;
